prog.go

```diff
@@ -143,6 +143,10 @@ func (p *Program) renderVars(agentRootDir string) {
 		arg = strings.Replace(arg, "$TaskId", strconv.Itoa(p.TaskId), -1)
 		p.Args[idx] = arg
 	}
+
+	p.Bin = strings.Replace(p.Bin, "$AgentRootDir", agentRootDir, -1)
```

Comment:
After PR https://github.com/openinx/huker/pull/32/files ,   this part has been changed ?  need to rebase ?